libs/sdk-ui-ext/src/internal/interfaces/Visualization.ts

```diff
@@ -332,6 +335,8 @@ export interface IVisualization {
         referencePoint: IReferencePoint,
         previousReferencePoint?: IReferencePoint,
     ): Promise<IExtendedReferencePoint>;
+
+    modifyInsightForDrillDown(source: IInsight, drillDownContext: IDrillDownContext): IInsight;
```

Comment:
I think this name is misleading. – it suggests the first argument will be modified, which is not true, right? How obout something like `getInsightWithDrillDownApplied`?

Also, please provide some form of TSDoc, so that the contract of this is described.

Reply:
Done